yt_dlp/YoutubeDL.py

```diff
@@ -2641,6 +2641,19 @@ def _fill_common_fields(self, info_dict, final=True):
             if final and info_dict.get('%s_number' % field) is not None and not info_dict.get(field):
                 info_dict[field] = '%s %d' % (field.capitalize(), info_dict['%s_number' % field])
 
+        multivalue_fields = {
+            'album_artist': 'album_artists',
+            'artist': 'artists',
+            'composer': 'composers',
+            'creator': 'creators',
+            'genre': 'genres',
+        }
+        for old_key, new_key in multivalue_fields.items():
+            if old_value := info_dict.get(old_key):
+                info_dict[new_key] = re.split(r', ?', old_value)
```

Comment:
Do we need the leniency? How about:

```suggestion
                info_dict[new_key] = old_value.split(', ')
```

Reply:
I thought that single comma is also a valid separator, but it seems like only comma with space is recognized by music players. So yes, we can remove this unnecessary leniency.